docs/en/latest/how-to-build.md

```diff
@@ -66,34 +67,50 @@ Please refer to: [Installing Apache APISIX with Helm Chart](https://github.com/a
 1. Create a directory named `apisix-2.11.0`.
 
   ```shell
-  mkdir apisix-2.11.0
+  $ APISIX_VERSION='2.11.0'
+  $ mkdir apisix-${APISIX_VERSION}
   ```
 
 2. Download Apache APISIX Release source package.
 
   ```shell
-  wget https://downloads.apache.org/apisix/2.11.0/apache-apisix-2.11.0-src.tgz
+  $ wget https://downloads.apache.org/apisix/2.11.0/apache-apisix-${APISIX_VERSION}-src.tgz
   ```
 
   You can also download the Apache APISIX Release source package from the Apache APISIX website. The [Apache APISIX Official Website - Download Page](https://apisix.apache.org/downloads/) also provides source packages for Apache APISIX, APISIX Dashboard and APISIX Ingress Controller.
 
 3. Unzip the Apache APISIX Release source package.
 
   ```shell
-  tar zxvf apache-apisix-2.11.0-src.tgz -C apisix-2.11.0
+  $ tar zxvf apache-apisix-2.11.0-src.tgz -C apisix-2.11.0
   ```
 
 4. Install the runtime dependent Lua libraries.
 
   ```shell
   # Switch to the apisix-2.11.0 directory
-  cd apisix-2.11.0
+  $ cd apisix-2.11.0
   # Create dependencies
-  make deps
+  $ make deps
   # Install apisix command
-  make install
+  $ make install
   ```
 
+## Step 2: Install ETCD
+
+This step is required if you have installed only Apache APISIX via RPM, Docker or source code but not ETCD.
+
+You can install ETCD via Docker or binary etc. The following command installs ETCD via binary.
+
+```shell
+$ ETCD_VERSION='3.4.13'
```

Comment:
We should use a command from install-dependencies.sh

Reply:
I have tried, but now install-dependencies.sh is not support to only install etcd.